libraries/AP_Mission/AP_Mission.h

```diff
@@ -619,6 +619,9 @@ class AP_Mission
     // returns true if the mission contains the requested items
     bool contains_item(MAV_CMD command) const;
 
+    // returns true if the mission has a terrain relative mission item
+    bool contains_terrain_relative(void);
```

Comment:
can we rename "constains_terrain_relative" to "contains_terrain_alt"?  No big deal of course but "terrain_alt" is what we use in Copter so it's more likely to show up in searches.

Reply:
Created a PR for that name change.